src/booking/Orders/OrdersTypes.ts

```diff
@@ -561,3 +562,10 @@ export interface ListParamsOrders {
 export interface UpdateSingleOrder {
   metadata: Order['metadata']
 }
+
+export interface AddServices {
```

Comment:
I don't wanna break a pattern that might exist here, but how about `AddServicesPayload` to make it clearer what this type is for.

Reply:
Hmm, we don't use the word payload anywhere, although it probably would make things clearer. The existing pattern is to name the type more or less after an endpoint (e.g. UpdateSingleOrder you can see above this one, CreateOrder...). I'll follow that for now but perhaps renaming them all to have "Payload" at the end would be an improvement to consider.